profiling-ffi/src/crashtracker/datatypes.rs

```diff
@@ -9,18 +9,30 @@ use ddcommon_ffi::{Error, Slice, StringWrapper};
 use std::ops::Not;
 use std::time::Duration;
 
+#[repr(C)]
+pub struct EnvVar<'a> {
+    key: CharSlice<'a>,
+    val: CharSlice<'a>,
+}
+
+#[repr(C)]
+pub struct CrashtrackerReceiverConfig<'a> {
+    pub args: Slice<'a, CharSlice<'a>>,
+    pub env: Slice<'a, EnvVar<'a>>,
+    pub path_to_receiver_binary: CharSlice<'a>,
+    /// Optional filename to forward stderr to (useful for logging/debugging)
```

Comment:
If we keep the requirement that stderr != stdout, might be useful to specify right in the header that they are assumed to be unique

Reply:
For now, we're opening them using `File::create`, so conflicting names would be an issue.

Reply:
👍